### DISCUSSION TITLE
Cylindrical and Spherical 3D Plots

### DISCUSSION THREAD

**Discussion by LeeSoomoon** [wxMaxima-developers/wxmaxima#1834] [General]
i study wxMaxima by youtube channel.(https://youtu.be/jvt3ijykTTI?si=xjWXqTLhmE0pnusf)

now i have a problem.

wxplot3d(5*(1-cos(t)), [t,0,2*%pi], [z,0,5],
 [gnuplot_postamble, "set mapping cylindrical"])$

his result is 
![image](https://github.com/wxMaxima-developers/wxmaxima/assets/146903197/7d64284b-dca3-43fc-88df-351b299928fe)

my result is 
![image](https://github.com/wxMaxima-developers/wxmaxima/assets/146903197/e83bb8fd-7ecb-468a-b43c-d1974c18dc18)

what is different?

**Comment by bolimbo** [comment]
Hi.  I got the same result as you.  As a workaround you could use: draw3d(cylindrical(5*(1-cos(t)),z,0,5,t,0,2*π))$.  I have wxMaxima version 23.05.1 on Win 11

**Reply by bolimbo** [reply]
To keep things inside wxMaxima and to get the plot inline:
`wxdraw3d(enhanced3d=true,cylindrical(5*(1-cos(t)),z,0,5,t,0,2*π));`
![image](https://github.com/wxMaxima-developers/wxmaxima/assets/149794338/58126ceb-b3bc-4b4a-ba85-3752a9c115f5)

**Comment by gunterkoenigsmann** [comment]
My question now is: Has something in maxima changed that causes the plot to be different to how it was when the video was made? And if the answer is yes: Is the change you see intended? If you replace `wxplot` by `plot` you might get an answer in the maxima-discuss mailing list at <https://sourceforge.net/p/maxima/mailman/maxima-discuss/>: Not that you have found a bug that needs reporting...
